src/assets/js/partials/digital-files.js

```diff
@@ -0,0 +1,85 @@
+    salla.lang.onLoaded(() => {
+      this.render();
+    });
+  }
+
+  render() {
+    if (!this.settings || Object.keys(this.settings).length === 0) {
+      this.innerHTML = "";
+      return;
+    }
+
+    const trans = (key) => salla.lang.get(key);
+    const formats = Array.isArray(this.settings.formats)
+      ? this.settings.formats.join(", ")
+      : this.settings.formats || "-";
+
+    this.innerHTML = `
+            <section class="bg-white p-5 rounded-md mb-5 last:mb-0">
+                <ul class="space-y-4">
+                    <li class="flex items-center justify-between">
+                        <div class="flex items-center gap-3">
+                            <i class="sicon-page"></i>
+                            <div class="text-gray-600 text-sm">${trans(
+                              "pages.products.number_of_files"
+                            )}</div>
+                        </div>
+                        <div class="text-gray-900">${
+                          this.settings.count || "-"
+                        }</div>
+                    </li>
+                    <li class="flex items-center justify-between">
+                        <div class="flex items-center gap-3">
+                            <i class="sicon-file-archive"></i>
+                            <div class="text-gray-600 text-sm">${trans(
+                              "pages.products.file_formats"
+                            )}</div>
+                        </div>
+                        <div class="text-gray-900 text-sm">${formats}</div>
+                    </li>
+                    <li class="flex items-center justify-between">
+                        <div class="flex items-center gap-3">
+                            <i class="sicon-calendar"></i>
+                            <div class="text-gray-600 text-sm">${trans(
+                              "pages.products.file_expiration_period"
+                            )}</div>
+                        </div>
+                        <div class="text-gray-900 text-sm">${
+                          this.settings.download_period || "-"
+                        }</div>
+                    </li>
+                    ${
+                      this.settings.access_new_files
+                        ? `
+                        <li class="flex items-center justify-between">
+                            <div class="flex items-center gap-3">
+                                <i class="sicon-rotate"></i>
+                                <div class="text-gray-600 text-sm">${trans(
+                                  "pages.products.free_access_to_new_files"
+                                )}</div>
+                            </div>
+                            <div class="text-gray-900 text-sm">
+                                <i class="sicon-check-circle text-lg"></i>
+                            </div>
+                        </li>
+                    `
+                        : ""
```

Comment:
Nice to Have:
I think it will better to be:
```
${this.accessFileList()}
//...

accessFileList(){
if(!this.settings.access_new_files){
return '';
}

return `the wanted dom`;
}
```